.github/workflows/tenzir.yaml

```diff
@@ -653,6 +651,8 @@ jobs:
         if: ${{ matrix.tenzir.name == 'Debian' }}
         run: |
           ./scripts/debian/install-dev-dependencies.sh
+          ./scripts/debian/install-aws-sdk.sh
```

Comment:
Let's scrap the Debian and Tenzir Plugins jobs. The same stuff is covered by the Docker build.